### DISCUSSION TITLE
error with windos dll when running, yarn prism migrate dev

### DISCUSSION THREAD

**Discussion by cristopherpds** [prisma/prisma#9912] [Q&A]
[print](https://imgur.com/a/2ElXBFH)

**Comment by janpio** [comment]
Why does the file not exist? Or does it - check the path manually please. Did you get an error message earlier?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
